Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -128,7 +134,7 @@ $radius: 15px;
                     <div>
                       {{ entry.pharmacy.address }}
                     </div>
-                    <div>
+                    <div v-if="entry.pharmacy.phoneType != 'F'">
```

Comment:
Lets use a constant or enum instead of a string

Reply:
Changed